packages/big-design/src/components/List/GroupSeparator/ListGroupSeparator.tsx

```diff
@@ -2,10 +2,12 @@ import React, { memo } from 'react';
 
 import { HR } from '../../Typography';
 
+import { StyledListItem } from './styled';
+
 export const ListGroupSeparator: React.FC<React.LiHTMLAttributes<HTMLLIElement>> = memo(() => (
-  <li tabIndex={-1} onMouseDown={preventFocus} aria-hidden={true}>
-    <HR />
-  </li>
+  <StyledListItem tabIndex={-1} onMouseDown={preventFocus} aria-hidden={true}>
+    <HR marginHorizontal="none" marginVertical="xSmall" />
```

Comment:
Do we need `marginHorizontal`? Looks like we don't

Reply:
removed